utils/lint_test_cases.sh

```diff
@@ -1,7 +1,7 @@
 #! /bin/bash
 
-SCT_SCYLLA_VERSION=4.2.0 ./sct.py lint-yamls -i '.yaml' -e 'multi-dc.yaml,multiDC,multidc,multiple-dc,rolling,docker,artifacts,private-repo,ics/long,scylla-operator,gce,jepsen'
+SCT_SCYLLA_VERSION=4.4.1 ./sct.py lint-yamls -i '.yaml' -e 'multi-dc.yaml,multiDC,multidc,multiple-dc,rolling,docker,artifacts,private-repo,ics/long,scylla-operator,gce,jepsen'
 
 SCT_SCYLLA_REPO='http://downloads.scylladb.com.s3.amazonaws.com/rpm/centos/scylla-2019.1.repo' ./sct.py lint-yamls -b gce -i 'multi-dc.yaml,multiDC,multidc,multiple-dc,rolling,artifacts,private-repo,gce,jepsen' -e 'docker'
 
-SCT_SCYLLA_VERSION=4.2.0 ./sct.py lint-yamls -b k8s-gce-minikube -i 'scylla-operator.*\.yaml'
+SCT_SCYLLA_VERSION=4.4.1 ./sct.py lint-yamls -b k8s-gce-minikube -i 'scylla-operator.*\.yaml'
```

Comment:
Append this change to the previous commit where you implement this line:
"""
fix(scy.py): implement lint-yamls in sct.py
"""

I do not see any reason to fix your own commit in the same PR.

Reply:
As you said a week ago we should not squash changes done with different purpose into one commit even if changes are done on the same file

Reply:
Ok, you distinguish these 2 this one, let it be so.